lib/led-matrix/src/LEDMatrix.cpp

```diff
@@ -0,0 +1,154 @@
+/* LED Matrix Source File */
+
+// naming with m_
+#include "LEDMatrix.h"
+
+#include "mbed.h"
+
```

Comment:
initially id put the comments in to help u understand the code. take em out if uve used em imo. alot of the code is self-documenting

Reply:
Ditto